cluster/common/velero/README.md

```diff
@@ -0,0 +1,70 @@
+# Velero Terraform Module
+
+This Terraform module implements Velero restore for a cluster migration scenario. You can use this Terraform module as part of any Terraform plan. It only depends on the existence of a Kubernetes cluster.
+
+Note: This module does not implement Velero backup, contributions are welcome.
```

Comment:
Do we need a module to implement Velero backup?  Maybe instead of asking for contributions, should we point them towards your fabrikate-definition component?

Reply:
Will do in a separate PR.